frontend/src/services/domain-service.ts

```diff
@@ -3,9 +3,10 @@ import { Resolver, ICache } from '@pwoosam/hashgraph-name-resolution-sdk';
 const domainResolver = new Resolver('hedera_main');
 domainResolver.init();
 
-export const getAccountIdFromDomain = async (domain: string): Promise<string | undefined> => {
-  return await domainResolver.resolveSLD(domain);
-}
+// Unused for now due to the state of .hbar domains
+// export const getAccountIdFromDomain = async (domain: string): Promise<string | undefined> => {
```

Comment:
We should add this back since this functionality still works, we're just opting out of using dev.hbar for nft explorer